packages/api-docs-builder/utils/parseSlots.ts

```diff
@@ -38,18 +47,18 @@ export default function parseSlots({
       if (tags.ignore) {
         return;
       }
-
-      slots[propertySymbol.name] = {
-        name: propertySymbol.name,
+      const slotName = propertySymbol.name;
+      slots[slotName] = {
+        name: slotName,
         description: getSymbolDescription(propertySymbol, project),
         default: tags.default?.text?.[0].text,
-        class: `.${muiName}-${propertySymbol.name}`,
+        class: classNames.includes(slotName) ? `.${muiName}-${slotName}` : '',
```

Comment:
Maybe it makes more sense for a missing class name to be null than an empty string, not sure

```suggestion
        class: classNames.includes(slotName) ? `.${muiName}-${slotName}` : null,
```

Reply:
`null` sounds better to me.